stdlib/2and3/tarfile.pyi

```diff
@@ -0,0 +1,172 @@
+## Stubs for tarfile
+
+from typing import (
+    Callable, ContextManager, IO, Iterable, List, Mapping, Optional, Type,
+    Union,
+)
+import sys
+from types import TracebackType
+
+
+ENCODING = ...  # type: str
+
+USTAR_FORMAT = ...  # type: int
+GNU_FORMAT = ...  # type: int
+PAX_FORMAT = ...  # type: int
+DEFAULT_FORMAT = ...  # type: int
+
+REGTYPE = ...  # type: bytes
+AREGTYPE = ...  # type: bytes
+LNKTYPE = ...  # type: bytes
+SYMTYPE = ...  # type: bytes
+DIRTYPE = ...  # type: bytes
+FIFOTYPE = ...  # type: bytes
+CONTTYPE = ...  # type: bytes
+CHRTYPE = ...  # type: bytes
+BLKTYPE = ...  # type: bytes
+GNUTYPE_SPARSE = ...  # type: bytes
+
+if sys.version_info < (3,):
+    TAR_PLAIN = ...  # type: int
+    TAR_GZIPPED = ...  # type: int
+
+def open(name: Optional[str] = ..., mode: str = ...,
+        fileobj: Optional[IO[bytes]] = ..., bufsize: int = ...,
+        *, format: Optional[int] = ..., tarinfo: Optional[TarInfo] = ...,
+        dereference: Optional[bool] = ...,
+        ignore_zeros: Optional[bool] = ...,
+        encoding: Optional[str] = ..., errors: str = ...,
+        pax_headers: Optional[Mapping[str, str]] = ...,
+        debug: Optional[int] = ...,
+        errorlevel: Optional[int] = ...) -> TarFile: ...
+
+
+class TarFile(ContextManager[TarFile], Iterable):
```

Comment:
I would prefer removing it for now. The same thing is done all over typeshed -- essentially ContextManager is recognized by `with` via duck typing, and `for` recognizes iterables the same way.